ingest-app/src/cmr/ingest/api/variables.clj

```diff
@@ -28,13 +28,21 @@
   [headers]
   (mt/extract-header-mime-type #{mt/json} headers "content-type" true))
 
+(defn- snake-case-data
```

Comment:
Hrm, this function appears in two places now (tagging is the other place); it's a special-case function for specific nested data structures, so it's a bit of an oddball ... should we put it in common-lib?